guides/blending/client-side-blending/README.md

```diff
@@ -0,0 +1,159 @@
+# Client-Side Blending
+
+When merging responses from different apps with [server-side blending](server-side-blending), the views are stacked on top of each other. In most cases, that's not what we want. Instead, we would like to compose elements in the views to make the result look like one page. That's what client-side blending does:
+
+![Blending comparison](/assets/SimpleBlendingDemo.PNG)
+
+In the example above, there are two apps, PetList and MedicalRecord. By default, they are stacked on top of each other. This makes it seem like they are not related, when they actually are. With client-side blending, we can move the table of examinations into the card from the PetList app and make it look like one coherent concept. In essence, we are **changing the layout but not the content** to combine apps that were not explicitly built to share screen. This is done without touching the source code of the individual apps.
+
+This ability of modifying the layout of views coming from different apps is crucial, especially when working with many apps. Without it, there would just be a stack of views with no meaningful layout.  
+
+## Layout and Content Separation
```

Comment:
> This is great, but we should use consistent jargon. I think that "presentation" can be changed to "layout" on page https://docs.starcounter.io/guides/web-apps/html-view-guidelines/

Fixed in https://github.com/Starcounter/Docs/commit/1d6fa02da77353e9149b2796c33faecc19bd3ef9